template/en/default/robots.txt.tmpl

```diff
@@ -2,7 +2,8 @@ User-agent: *
 Disallow: [% basepath FILTER none %]
 Crawl-delay: 30
 
-[% IF NOT Bugzilla.localconfig.urlbase.matches("bugzilla-dev") %]
+[% USE Bugzilla %]
+[% IF Bugzilla.localconfig.urlbase.match("bugzilla.mozilla.org") %]
```

Comment:
This should be an exact string match on production's `urlbase` rather than a substring regex match.